.github/workflows/check_style.yaml

```diff
@@ -0,0 +1,24 @@
+name: Style Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install requirements
+      run: pip install -r requirements.txt
+    - name: Check Code format
+      working-directory: Tools
+      run: |
+        ls -la
+        chmod +x ./check_code_style.sh
```

Comment:
git tracks executable permissions, so you can just commit it

Reply:
Thanks. Done!